src/Microsoft.AspNetCore.SystemWebAdapters.SessionState/Serialization/UnknownSessionKeyException.Shared.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.SessionState.Serialization;
+
+public class UnknownSessionKeyException : Exception
```

Comment:
```suggestion
public class UnknownSessionKeyException : InvalidOperationException
```

Does this new exception type really need to be public? Do we actually expect people to catch it? Or can we use InvalidOperationException instead and put the keys in the message?

We want to avoid introducing new APIs as part of the adapters when people are expected to remove the adapters later.